addons/web/static/src/legacy/js/core/cookie_utils.js

```diff
@@ -21,21 +21,40 @@ return {
         }
         return "";
     },
+    /**
+     * Check if cookie can be written.
+     *
+     * @param {String} type the type of the cookie
+     * @param {String} name the name of the cookie
+     * @returns {boolean}
+     */
+    checkCookie(type, name) {
```

Comment:
Nitpicking: The name is a bit confusing, it sounds like it is actually checking the cookie value somehow.
You can ignore this remark and mark as done if you think it is fine as is, otherwise a better name would be great :+1: 
Something with `can` or `allow` maybe

Edit: or simply as the python name which is way clearer: isAllowedCookies